examples/binding-patterns-in-match-statement/binding_patterns_in_match_statement.bal

```diff
@@ -0,0 +1,51 @@
+import ballerina/io;
+
+type Pair record {
+    int x;
+    int y;
+};
+
+type Rec record {
+    Pair p;
+};
+
+function matchFn1(Pair pair) {
+    match pair {
+        // The binding pattern below matches mappings that contain at least the fields `x` and `y`.
+        // The values of these fields can be accessed via the variables `x` and `y` within this block.
+        var {x, y} => {
+            io:println(x, ", ", y);
+        }
+    }
+}
+
+function matchFn2(Pair pair) {
+    match pair {
+        // The binding pattern below also has a rest binding pattern to capture the additional fields
+        // that may be defined in `pair` since it is an open record.
```

Comment:
```suggestion
        // that may be defined in the `pair` since it is an open record.
```

Reply:
Here, `pair` is used as a reference, and is referring to a variable name. So we don't need "the".

If we are using "the" we have to change it to something like
```suggestion
        // that may be specified in the open record value assigned to the `pair` variable.
```

Reply:
Added the latter.